src/simulation/elements/INVIS.cpp

```diff
@@ -6,7 +6,7 @@ Element_INVIS::Element_INVIS()
 	Name = "INVS";
 	Colour = PIXPACK(0x00CCCC);
 	MenuVisible = 1;
-	MenuSection = SC_SENSOR;
+	MenuSection = SC_SPECIAL;
```

Comment:
INVS is not really sensor - yes, it changes it's state while under pressure, but detecting stuff isn't it's main goal - there's PSNS for detecting pressure.